net-libs/tox/tox-9999.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools eutils git-r3 user systemd
+
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/irungentoo/toxcore.git
+	git://github.com/irungentoo/toxcore.git"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS=""
+IUSE="+av daemon log log-error log-info log-trace log-warn +no-log ntox static-libs test"
+
+REQUIRED_USE="^^ ( no-log log log-trace log-info log-warn log-error )"
+
+RDEPEND="
+	>=dev-libs/libsodium-0.6.1[urandom,asm]
+	daemon? ( dev-libs/libconfig )
+	av? ( media-libs/libvpx
+		media-libs/opus )
+	ntox? ( sys-libs/ncurses )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	test? ( dev-libs/check )"
+
+src_prepare() {
+	epatch_user
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable log) \
+		$(usex log-trace "--enable-log --with-log-level=TRACE" "") \
+		$(usex log-info "--enable-log --with-log-level=INFO" "") \
+		$(usex log-warn "--enable-log --with-log-level=WARNING" "") \
+		$(usex log-error "--enable-log --with-log-level=ERROR" "") \
+		$(use_enable av) \
+		$(use_enable test tests) \
+		$(use_enable ntox) \
+		$(use_enable daemon) \
+		$(use_enable static-libs static)
+}
+
+src_install() {
+	default
+	if use daemon; then
+		newinitd "${FILESDIR}"/initd tox-dht-daemon
+		newconfd "${FILESDIR}"/confd tox-dht-daemon
+		insinto /etc
+		doins "${FILESDIR}"/tox-bootstrapd.conf
+		systemd_dounit "${FILESDIR}"/tox-bootstrapd.service
+	fi
+
+	prune_libtool_files
+}
+
+pkg_postinst() {
+	if use daemon; then
+		enewgroup ${PN}
+		enewuser ${PN} -1 -1 -1 ${PN}
+		ewarn "Backwards compatability with the bootstrap daemon"
```

Comment:
Maybe show this only if `[[ -f ${EROOT}var/lib/tox-dht-bootstrap/key ]]` exists.